caddytest/integration/caddyfile_adapt/global_options.txt

```diff
@@ -7,7 +7,6 @@
 	storage file_system {
 		root /data
 	}
-	acme_ca https://example.com
```

Comment:
Should leave this in, actually. That should continue to work.

Reply:
I can put it back, I removed it so that I had something testing when the option wasn't specified at all,